module/modmanager/manager.go

```diff
@@ -1119,15 +1121,49 @@ func (mgr *Manager) FirstRun(ctx context.Context, conf config.Module) error {
 	for key, val := range moduleEnvironment {
 		cmd.Env = append(cmd.Env, key+"="+val)
 	}
-	cmdOut, err := cmd.CombinedOutput()
 
-	resultLogger := logger.With("path", firstRunPath, "output", string(cmdOut))
+	stdOut, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+	stdErr, err := cmd.StderrPipe()
 	if err != nil {
-		resultLogger.Errorw("command failed", "error", err)
 		return err
 	}
 
-	resultLogger.Infow("command succeeded")
+	scanOut := bufio.NewScanner(stdOut)
+	go func() {
+		for scanOut.Scan() {
+			logger.Infow("got stdio", "output", scanOut.Text())
+		}
+		// This scanner keeps trying to read stdio until the command terminates,
+		// at which point the stdio pipe handle is no longer available. This sometimes
+		// results in an `os.ErrClosed`, which we discard.
+		if err := scanOut.Err(); err != nil && !errors.Is(err, os.ErrClosed) {
+			logger.Errorw("error scanning stdio", "error", err)
+		}
+	}()
+	scanErr := bufio.NewScanner(stdErr)
+	go func() {
+		for scanErr.Scan() {
+			logger.Warnw("got stderr", "output", scanErr.Text())
+		}
+		// This scanner keeps trying to read stderr until the command terminates,
+		// at which point the stderr pipe handle is no longer available. This sometimes
+		// results in an `os.ErrClosed`, which we discard.
+		if err := scanErr.Err(); err != nil && !errors.Is(err, os.ErrClosed) {
+			logger.Errorw("error scanning stderr", "error", err)
+		}
+	}()
+	if err := cmd.Start(); err != nil {
+		logger.Errorw("failed to start", "error", err)
+		return err
+	}
+	if err := cmd.Wait(); err != nil {
+		logger.Errorw("failed", "error", err)
+		return err
+	}
+	logger.Infow("succeeded")
```

Comment:
```suggestion
	logger.Info("first run script succeeded")
```

Reply:
so I deliberately removed this from the log message since "first run" is indicated by the sublogger name: https://github.com/viamrobotics/rdk/pull/4475/commits/cfdd5df3afdb899384439d421f5e9acd701218a3

Reply:
on second thought, maybe it's fine to be a teensy-bit more verbose - will update with your suggestion

Reply:
the main log I wanted to trim "first run" from was the stdio + stderr logs, since that distracted from the subprocess output IMO

Reply:
ya I think getting rid of those from the stdio + stderr logs definitely makes sense